gradle/dependencies.gradle

```diff
@@ -76,7 +76,7 @@ def support = [
     arch: [
         lifecycle: [
             compiler: "android.arch.lifecycle:compiler:${versions.archComponents}",
-            extensions: "android.arch.lifecycle:extensions:${versions.archComponents}",
+            common: "android.arch.lifecycle:extensions:${versions.archComponents}",
```

Comment:
Shouldn't that be `android.arch.lifecycle:common`?

Reply:
Derp, you're right

Reply:
7d16288